### ISSUE TITLE
Feditext crashes on resume/autorefresh

### ISSUE THREAD

**Issue by mrbinsk** [feditext/feditext#235]
Feditext quiet crashes when switching back to it after using another app. No display error, but ability to send extra info. 

1. Use Feditext
2. Switch to another app or turn off screen
3. Resume Feditext
4. Feditext intermittently exits without display error (sent more info when prompted)

**Expected behavior**
Feditext stays open for usage

 - Device: iPhone 14 Pro Max
 - OS: 16.6
 - App Version 1.7.2 (73)

**Additional context**
Seems to have been introduced in 1.7.2 (73). Is intermittent and does not reproduce every time, but I wanted to submit a bug along with my additional data submission.

**Comment by VyrCossont** [comment]
@mrbinsk Thanks for the report; I'll try to find the extra crash data in Apple's dev portal. Is this happening on infosec.town, and have you observed it on any other servers?

**Comment by mrbinsk** [comment]
@VyrCossont Yes, this is against infosec.town. I have not been using fedia.social, another Firefish instance, to confirm that same behavior, but I can try and I will update if it happens there as well. This hasn't consistently reproed with any specific actions, but it's happened enough times where I don't think it's a one-off.

**Comment by VyrCossont** [comment]
@mrbinsk right on, thanks.

**Comment by bdube** [comment]
@VyrCossont if the crash reports aren't in your portal, I'll download them all and post details when I can.

**Comment by bdube** [comment]
I downloaded the zip archives for the crash reports, but they contain very little information compared to what you get by clicking the Open in Xcode button in the dev portal.
